changelog_unreleased/javascript/pr-7875.md

```diff
@@ -0,0 +1,27 @@
+#### Add --embedded-language-formatting={auto,off} option ([#7875](https://github.com/prettier/prettier/pull/7875) by [@bakkot](https://github.com/bakkot))
+
+When Prettier identifies cases where it looks like you've placed some code it knows how to format within a string in another file, like in a tagged template in JavaScript with a tag named `html` or in code blocks in Markdown, it will by default try to format that code.
+
+Sometimes this behavior is undesirable, since it can change the behavior of your code. This option allows you to switch between the default behavior (`auto`) and disabling this feature entirely (`off`). It applies to all languages where Prettier recognizes embedded code, not just JavaScript.
+
+<!-- prettier-ignore -->
+```js
+// Input
+html`
+<p>
+I am expecting this to come out exactly like it went in.
+`;
+
+// using --template-formatting=auto (or omitting this option)
```

Comment:
Typo `--template-formatting=` to `--embedded-language-formatting`

Reply:
Thanks, fixed.